src/util.h

```diff
@@ -28,6 +28,8 @@
 #include <boost/date_time/posix_time/posix_time_types.hpp>
 #include <boost/thread.hpp>
 #include <boost/thread/condition_variable.hpp>
+#include <boost/iostreams/filtering_stream.hpp>
+#include <boost/iostreams/filter/newline.hpp>
```

Comment:
Can we put these into util.cpp (so they're not included into every file that needs util.h)?

Reply:
Sure.

Reply:
We actually need to break util apart like bitcoin. But that is for another day...

Reply:
Yep. One day...